cask.el

```diff
@@ -156,6 +156,16 @@ Slots:
 `column' Column number on the `line'."
   line column)
 
+;; Specializations of package-build classes and methods to define a
```

Comment:
This stuff should ideally go into https://github.com/melpa/package-build to avoid having it separately. The failing travis builds for emacs 24.3 could possibly be solved by moving the code there. Just a wild guess really since I don't know `eieio` at all.
@tarsius: What do you say about this? Does it make sense to suppor directory based packages in `package-build`?